source/puppet/5.3/hiera_automatic.md

```diff
@@ -6,20 +6,20 @@ title: "Looking up data with Hiera"
 [data_type]: ./lang_data_type.html
 [editing_data]: ./hiera_merging.html
 
-{:.Concept}
-## Class parameters 
-  
+{:.concept}
+## Class parameters
+
 Puppet looks up the values for class parameters in Hiera, using the fully-qualified name of the parameter (`myclass::parameter_one`) as a lookup key.
 
 Most classes need configuration, and you can specify them as parameters to a class as this will lookup the needed data if not directly given when the class is included in a catalog. There are several ways Puppet sets values for class parameters, in this order:
 
-1. If you’re doing a resource-like declaration, Puppet uses parameters that are explicitly set (if explicitly setting undef, a looked up value or default will be used).
```

Comment:
Removed "smart" quotes from the content. These won't always cause problems, but we should let the publishing tools handle typography.